articles/automated-fake-news-detection/index.md

```diff
@@ -0,0 +1,130 @@
+Natural Language Processing enables computers to understand natural/human language and respond appropriately. Hence, there are two aspects involved:
+· Natural Language Understanding
+· Natural Language Generation
+The two approaches to fake news detection are:
+· Machine Learning approach
+· Deep Learning approach
+
+#### Machine Learning approach
+
+Machine learning refers to giving computers ability to learn without explicitly being programmed.
+A machine learning approach uses machine learning algorithms to detect misinformation. Examples of these algorithms include:
+**Naïve Bayes:** uses probabilistic approaches based on Bayes theorem. This algorithm is often used for text classification.
+**Decision tree:** a supervised learning algorithm that has a tree-like flow. It helps in decision making. A useful algorithm for both classification and regression tasks.
+Random forest: simply a combination of decision trees.
+**Support Vector Machine:** a supervised learning algorithm. It examines data for classification and regression analysis. It classifies data into two categories.
+**Logistic regression:** contrary to the name, it is a classification algorithm used to estimate discrete values.
+**K-nearest-neighbour:** a simple algorithm that is used for both classification and regression tasks. Though it is more widely used for classification problems.
+
+Datasets are used to refine the algorithms. These datasets may be split as training data or test data.
+I have come across a lot of research where a system combines various machine learning algorithms and data mining. Often carried out on social media platforms, especially Twitter data. For example, a model may combine machine learning, through Naïve Bayes, Support Vector Machine, and Natural Language Processing to detect fake news. The accuracy of the results is usually determined by the combinations of models used and the datasets involved. A combination of available toolkits with Bayesian learning may be used to develop a fake news detector. These toolkits include Textblob, Natural Language and SciPy.
+
+But a challenge exists with some of these traditional machine learning approaches. They treat fake news detection as a binary classification task. These models alone struggle to contextualize text data. They need structured/labelled data. In fact, machine learning models struggle to solve complex queries with huge amounts of data. This is where deep learning models come in.
+
+#### Deep Leaning Approach
+
+Deep learning algorithms function similar to machine learning algorithms. But there is a key difference. Deep learning algorithms have layers that interpret data differently. Artificial neural networks refer to a network of such algorithms.
+Purely deep learning perspectives towards fake news have been explored in many works. Links to some at the end of the article.
```

Comment:
explored in many works. Links to some at the end -> explored in many works. Links to some [articles/code/what is it a link to ] at the end